src/ncurses_ui.cc

```diff
@@ -686,9 +688,10 @@ Optional<Key> NCursesUI::get_next_key()
                       case '7': f = alt_ctrl; break;
                       case '8': f = shift_alt_ctrl; break;
                   }
-                  if ( f == NULL )
+                  if ( f == nullptr )
```

Comment:
If you want to simplify this further:

```c++
if (f)
```

There's a [boolean operator](https://en.cppreference.com/w/cpp/utility/functional/function/operator_bool) for that. Also no spaces between the parenthesis.